packages/p2p/src/components/file-uploader-component/file-uploader-component.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FileDropzone, Icon, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize } from 'Components/i18next';
+
+const FileUploaderComponent = ({
+    accept,
+    hover_message,
+    max_size,
+    multiple = false,
+    onClickClose,
+    onDropAccepted,
+    onDropRejected,
+    upload_message,
+    validation_error_message,
+    value,
+}) => {
+    const getUploadMessage = () => {
```

Comment:
Can we move this out of the `FileUploaderComponent`?

Reply:
included in file-uploader-component itself as it is part of the component